extra_data/reader.py

```diff
@@ -1310,12 +1310,17 @@ def RunDirectory(path, include='*', file_filter=locality.lc_any):
         Function to subset the list of filenames to open.
         Meant to be used with functions in the extra_data.locality module.
     """
-    files = [f for f in os.listdir(path) if f.endswith('.h5')]
+    files = [f for f in os.listdir(path) if f.endswith('.h5') and f != 'overview.h5']
     files = [osp.join(path, f) for f in fnmatch.filter(files, include)]
-    files = file_filter(files)
-    if not files:
+    sel_files = file_filter(files)
+    if not sel_files:
         raise Exception("No HDF5 files found in {} with glob pattern {}".format(path, include))
 
+    if _use_voview and (sel_files == files):
```

Comment:
If some files were filtered out by the locality filter, this won't use the virtual overview file, to avoid accidentally trying to access data from tape when we've asked for only data on disk.

However, if you filter files with e.g. `include='*AGIPD*'`, we'll still read a virtual overview file if possible. This could be unexpected in some situations - e.g. if you use `.train_from_id()` to get data from all sources, you'll see ones that weren't previously included. But the primary reason for the `include=` parameter is to open a run *faster*, and using the virtual overview file should be a fast option if it exists.